client/src/views/users/GreetUserModal.vue

```diff
@@ -500,8 +503,10 @@ async function nextStep(): Promise<void> {
     const result = await greeter.value.getClaimRequests();
     waitingForGuest.value = false;
     if (result.ok) {
-      guestInfoPage.value.fullName = greeter.value.requestedHumanHandle?.label;
-      guestInfoPage.value.email = greeter.value.requestedHumanHandle?.email;
+      if (guestInfoPageRef.value) {
```

Comment:
Probably better to condense the two ifs so that the error is also displayed and we don't go forward if we don't have the ref.